unlock-app/src/services/graphService.ts

```diff
@@ -55,9 +55,7 @@ export class GraphService {
     const query =
       filterKey === 'owner' ? keyHoldersByLocks() : keyholdersByKeyIdQuery()
 
-    const keyId = !isNaN(parseInt(`${search}`))
```

Comment:
We need to show an indication to the user when the number is not valid, otherwise the user does not know what is going on.
Maybe add validation to the form field rather than here?

Reply:
Need it also here, but fine to add also a visible error.
Otherwise we have to add a `Confirm/Search` button that's run the query when pressed only when validation passes